static/index.html

```diff
@@ -80,7 +80,7 @@
             xmlns:xlink="http://www.w3.org/1999/xlink"
             viewBox="0 0 490 490"
             xml:space="preserve"
-            style="cursor: pointer; padding: 10px"
+            style="display: none; cursor: pointer; padding: 10px"
```

Comment:
Seems like these inline styles would be better implemented as classes. Would you mind handling that?

Reply:
No worries and yeah I tested it by just passing an empty array then an array with two objects but I'll grab a screenshot for qa